src/Sulu/Bundle/ContactBundle/Util/IdsHandlingTrait.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactBundle\Util;
+
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+
+/**
+ * Provides function to sort array by ids array.
+ */
+trait IdsHandlingTrait
```

Comment:
Not sure if there would be a better alternative than using a trait, maybe a service? Guess I will have another look at that tomorrow, maybe somebody has a better idea till then :smile: